ui/app/models/pki/tidy.js

```diff
@@ -131,10 +149,21 @@ export default class PkiTidyModel extends Model {
   tidyRevokedCerts;
 
   get allGroups() {
-    const groups = [{ autoTidy: ['enabled', 'intervalDuration'] }, ...this.sharedFields];
+    const groups = [
+      { autoTidy: ['enabled', 'intervalDuration', 'minStartupBackoffDuration', 'maxStartupBackoffDuration'] },
+      ...this.sharedFields,
+    ];
     return this._expandGroups(groups);
   }
 
+  // fields that are specific to auto-tidy, which should only show up when enabled.
```

Comment:
thank you for this comment! 🤩  If auto tidy is enabled, then disabled, will these settings be preserved? If so, then I might suggest renaming the getter to `autoTidyConfigFields` since the enabled toggle is more for display only purposes in the UI

Reply:
Will update to the proposed name, yes the values are preserved if auto-tidy is enabled/disabled.